ai21/models/assistant/run.py

```diff
@@ -0,0 +1,41 @@
+from typing import Literal, Any, List
```

Comment:
Since everything you added is under beta, I suggest to also move the packages themselves to be under beta package

e.g. folder structure
- beta 
  - assistant.py
  - threads.py
  - ...

Reply:
`conversational_rag` is also under beta but isn't listed under `beta` directory...